src/core/ext/lb_policy/grpclb/grpclb.c

```diff
@@ -804,8 +805,11 @@ static char *get_lb_uri_target_addresses(grpc_exec_ctx *exec_ctx,
       char *addr_str;
       GPR_ASSERT(grpc_sockaddr_to_string(
                      &addr_str, &addresses->addresses[i].address, true) > 0);
-      targets_info_entries[addr_index] = targets_info_entry_create(
-          addr_str, addresses->addresses[i].balancer_name);
+      if (addresses->addresses[i].balancer_name != NULL) {
```

Comment:
Who said it's okay for this to be NULL?  I think that should not be allowed, so we should not have to handle it here.